src/components/ebay-line-chart/component.ts

```diff
@@ -321,7 +322,7 @@ class LineChart extends Marko.Component<Input> {
             backgroundColor: tooltipBackgroundColor, // sets tooltip background color
             borderWidth: 0, // hide the default border stroke
             borderRadius: 10, // set the border radius of the tooltip
-            outside: true, // used to render the tooltip outside of the main SVG element
+            outside: component.input.outside || true, // used to render the tooltip outside of the main SVG element
```

Comment:
Will this not _always_ be true? Maybe it is meant to be `component.input.outside !== false`?

Reply:
Oops, thanks!

Reply:
Sure, that's the same as `component.input.outside !== false`